src/modules/esl-media/core/esl-media.ts

```diff
@@ -300,16 +302,9 @@ export class ESLMedia extends ESLBaseElement {
     condition: ($this: ESLMedia) => $this.fillModeEnabled
   })
   protected _onResize(): void {
-    if (!this._provider) return;
-    if (this.fillModeEnabled && this.actualAspectRatio > 0) {
-      let stretchVertically = this.offsetWidth / this.offsetHeight < this.actualAspectRatio;
-      if (this.fillMode === 'inscribe') stretchVertically = !stretchVertically; // Inscribe behaves inversely
-      stretchVertically ?
-        this._provider.setSize(this.actualAspectRatio * this.offsetHeight, this.offsetHeight) : // h
-        this._provider.setSize(this.offsetWidth, this.offsetWidth / this.actualAspectRatio);   // w
-    } else {
-      this._provider.setSize('auto', 'auto');
-    }
+    const {actualAspectRatio} = this;
```

Comment:
```suggestion
    if (!this._provider) return;
    const {actualAspectRatio} = this;
```
Still primary condition to do anything